### PR TITLE
Store handle multiple records

### PR DESCRIPTION
**Pull request by stroem**

### CODE REVIEW

**Comment by stroem** [comment]
Test is not fixed yet

**Comment by skiarn** [comment]
Looks good but you need to Rollback the tx if error occurs

File: eventsource/stores/sqlstore/store.go [hunk 1]
```diff
@@ -27,22 +27,38 @@ func New(db *sql.DB, tableName string) eventsource.Store {
...
 	if err != nil {
-		return
+		return err
+	}
+	defer stmt.Close()
+
+	for _, record := range records {
+		_, err = stmt.ExecContext(ctx, record.AggregateID, record.SequenceID, record.Timestamp, record.UserID, record.Type, record.Data)
+		if err != nil {
+			return err
```
Comment (skiarn) [inline eventsource/stores/sqlstore/store.go]: I think you always want to rollback tx on error.

File: eventsource/stores/sqlstore/store.go [hunk 2]
```diff
@@ -27,22 +27,38 @@ func New(db *sql.DB, tableName string) eventsource.Store {
...
 
 // SaveWithContext ...
-func (store *store) SaveWithContext(ctx context.Context, record eventsource.Record) (err error) {
-	stmt, err := store.db.PrepareContext(ctx, fmt.Sprintf(saveSQL, store.tablename))
+func (store *store) SaveWithContext(ctx context.Context, records ...eventsource.Record) (err error) {
+	tx, err := store.db.BeginTx(ctx, nil)
 	if err != nil {
 		return
 	}
-	defer stmt.Close()
```
Comment (skiarn) [inline eventsource/stores/sqlstore/store.go]: I think Rollback and commit will automatically call stmt.Close for all statements in that transaction tx, see https://golang.org/src/database/sql/sql.go

**Pull request merged**